lib/evented/index.js

```diff
@@ -19,7 +19,37 @@ var ArrayProto = Array.prototype;
 var nativeIndexOf = ArrayProto.indexOf;
 var slice = ArrayProto.slice;
 
-function bind(event, fn) {
+/**
+ * Initialize a custom error handler or default to the BV Loader error reporting
+ * regardless of application. This will automatically be called at the first
+ * invocation of the ".on" method, and can be 
+ * event
+ * @param {Function} fn - the custom callback to be invoked whenever an error is
+ *  detected as the result of a trigger event
+ */
+function setErrorHandler(fn) {
+  this.errorHandler = function(e) {
+    global.console.error(e);
```

Comment:
I feel like we should only log if there is NOT a custom error handler.  Users of this library should be able to determine what gets logged to their console.